### PR TITLE
examples/grpc-client: clean up the example and add tlsClientCA to ConfigMap.

### PR DESCRIPTION
**Pull request by rithujohn191**

### CODE REVIEW

File: examples/grpc-client/client.go [hunk 1]
```diff
@@ -42,21 +42,7 @@ func newDexClient(hostAndPort, caPath, clientCrt, clientKey string) (api.DexClie
...
-	if *clientCrt == "" || *caCrt == "" || *clientKey == "" {
-		log.Fatal("Please provide CA & client certificates and client key. Usage: ./client --ca-crt=<path ca.crt> --client-crt=<path client.crt> --client-key=<path client key>")
-	}
-
-	client, err := newDexClient("127.0.0.1:5557", *caCrt, *clientCrt, *clientKey)
-	if err != nil {
-		log.Fatalf("failed creating dex client: %v ", err)
-	}
-
+func testPassword(cli api.DexClient) error {
```
Comment (ericchiang) [inline examples/grpc-client/client.go]: "testPassword" seems like an odd name for this function. It makes it sound like a unit test. Maybe just "createPassword"?

Comment (rithujohn191) [inline examples/grpc-client/client.go]: done

**Comment by ericchiang** [comment]
Note that I didn't test this manually. Maybe we should have travis do that? (might be overkill)

**Comment by rithujohn191** [comment]
I tested this change by running the grpc example.

**Comment by rithujohn191** [comment]
I could create a separate issue to have Travis run examples.

**Comment by ericchiang** [comment]
@rithujohn191 don't worry about it. I think it's fine for now.

**Pull request merged**